.github/workflows/test.yml

```diff
@@ -393,4 +393,4 @@ jobs:
         uses: "./.github/actions/bundle_size_test"
         with:
           cwd: ${{ env.PROJECT_DIR }}
-          config: '{"build/main.js": "10kB", "build/vendor.js": "390kB"}'
+          config: '{"build/main.js": "59kB", "build/vendor.js": "390kB"}'
```

Comment:
since CI is now generating the right project. we are measuring the chakra-storefront generated app vs typescript-minimal previously